src/views/AfhandelingView.vue

```diff
@@ -6,8 +6,29 @@
       :to="{ name: 'contactmoment' }"
       >terug</router-link
     >
-    <section class="layoutcontainer-small">
-      <contactmoment-afhandel-form />
+    <section>
```

Comment:
de markup is volgens mij niet helemaal logisch. er is een main, met een heading, maar zonder inhoud die bij die heading hoort, want direct daaronder begint een section. Ofwel die section heeft geen waarde, ofwel de heading hoort binnen de section, lijkt me. Als het puur is voor styling doeleinden, zou ik eerder een div verwachten.

Reply:
todo: vervangen door div